language/move-lang/src/expansion/translate.rs

```diff
@@ -120,31 +152,42 @@ impl Context {
 //**************************************************************************************************
 
 pub fn program(prog: P::Program, sender: Option<Address>) -> (E::Program, Errors) {
-    let mut context = Context::new();
+    let module_members = {
+        let mut members = BTreeMap::new();
+        all_module_members(&mut members, sender, &prog.lib_definitions);
+        all_module_members(&mut members, sender, &prog.source_definitions);
+        members
```

Comment:
Nit: Why not just return the context here instead of passing it back then creating it.